### PR TITLE
update trade api and add is_trade_simulated

### PR DESCRIPTION
**Pull request by GuillaumeDSM**

### CODE REVIEW

File: octobot_trading/api/trades.py [hunk 1]
```diff
@@ -30,4 +29,8 @@ def get_total_paid_trading_fees(exchange_manager) -> dict:
 
 
 def get_trade_exchange_name(trade) -> str:
-    return get_order_exchange_name(trade.order)
+    return trade.exchange_manager.get_exchange_name()
+
+
+def is_trade_simulated(trade) -> bool:
```
Comment (Herklos) [inline octobot_trading/api/trades.py]: Does order have a `simulate` attribute ?

Comment (GuillaumeDSM) [reply octobot_trading/api/trades.py]: Nope

Comment (Herklos) [reply octobot_trading/api/trades.py]: Ok, do you think a `simulate` attribute have to be added into `Order` and `Trade` ?

Comment (GuillaumeDSM) [reply octobot_trading/api/trades.py]: If we want to save trades between bot instances, I think it's better to add it yep

**Comment by GuillaumeDSM** [comment]
done in https://github.com/Drakkar-Software/OctoBot-Trading/pull/95

**Pull request closed without merging**